src/app_service/service/transaction/dto.nim

```diff
@@ -31,7 +31,8 @@ type
     RemoteDestructCollectible = "RemoteDestructCollectible"
     BurnCommunityToken = "BurnCommunityToken"
     DeployOwnerToken = "DeployOwnerToken"
-    SetSigner = "SetSigner"
+    SetSignerPublicKey = "SetSignerPublicKey"
```

Comment:
@noeliaSD I added you because of this line. It was added by you, but I don't see any `SetSigner` identification on the statusgo side, guess it should be `SetSignerPublicKey`.

Reply:
SetSignerPublicKey is correct.